src/app/Middleware/version.cs

```diff
@@ -46,7 +46,10 @@ public static IApplicationBuilder UseVersion(this IApplicationBuilder builder)
                         }
 
                         // build and cache the json string
-                        string json = "{ " + $"\"apiVersion\": \"{swaggerVersion}\", \"appVersion\": \"{Middleware.VersionExtensions.Version}\"" + " }";
+                        string json = "{ " +
```

Comment:
you should be able to escape the { and } and put this in one format string

having multiple format strings and concatenating strings with + is inefficient

I would either put in one format string or use string builder

Reply:
done